src/test/java/lessonSevenPythagoreanTriple/PythagoreanTripleTest.java

```diff
@@ -0,0 +1,25 @@
+package lessonSevenPythagoreanTriple;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PythagoreanTripleTest {
+
+    @Test
+    void shouldBePythagorean1() {
```

Comment:
Dodawanie 1, 2, 3... w nazwach czegokolwiek nie jest dobrym pomysłem. Sprawiło to, że automatycznie pomyślałem, że te testy testują to samo. A patrząc na sam kod to tak nie jest.

Reply:
w zasadzie sprawdzają one to samo, jedynie podając w różnej kolejności boki. Usuwam w takim razie 2 testy.